studio/schemas/documents/benefit.ts

```diff
@@ -1,27 +1,45 @@
-import { defineType } from 'sanity';
-import { richText, title } from '../fields/text';
+import { defineField, defineType } from "sanity";
+import { richText, title } from "../fields/text";
 
 export const benefitId = "benefit";
+export const benefitTypeId = "benefitType";
+
+const BENEFIT_TYPES = [
+  { title: "Gadgets", value: "gadgets" },
+  { title: "Bonus", value: "bonus" },
+  { title: "Pension", value: "pension" },
+  { title: "Salary Growth", value: "salaryGrowth" },
+];
+
+const benefitType = defineField({
+  name: benefitTypeId,
+  type: "string",
+  title: "Benefit Type",
+  description:
+    "Choose the type of benefit, or leave blank for generic benefits. Some benefit types include visual graphs that will be displayed together with the text.",
+  options: {
+    list: BENEFIT_TYPES,
+    layout: "dropdown",
+  },
+});
 
 const benefit = defineType({
   name: benefitId,
   type: "document",
   title: "Benefit",
-  fields: [
-    title,
-    richText
-  ],
+  fields: [benefitType, title, richText],
   preview: {
     select: {
       title: title.name,
+      type: benefitType.name,
     },
-    prepare({ title }) {
+    prepare({ title, type }) {
       return {
         title,
-        subtitle: "Benefit",
+        subtitle: `Benefit${type ? ` / ${BENEFIT_TYPES.find((o) => o.value === type)?.title ?? "Unknown benefit type"}` : ""}`,
```

Comment:
Ok, so this might be changed with the initialValue and required ruleset of BenefitType.. but I'd also like to simplify this for easier understanding of what's happening. 

```
const benefitType = BENEFIT_TYPES.find((o) => o.value === type)?.title;
const subtitle = type ? benefitType :  "Unknown benefit type";  // though I'm not sure this is needed now that BenefitType has initialValue
```

```suggestion
        subtitle: subtitle,
```

Reply:
"Unknown benefit type" was mostly intended for catching strings not matching any of the current benefit types (which could change over time).

Probably a good idea to split it up :+1: but the suggested change is missing the "Benefit / " prefix. Do you not want that?

Reply:
Hm, no we already know it'll be added to the list of benefits, so it's kind of redundant

Reply:
Ok, but the subtitle should probably not be "None" for benefits with type "None"? Maybe not have a subtitle for those?

Reply:
We could do "Type of benefit: Gadgets" as subtitle but that would also set it to "Type of Benefit: none" ... 🤔

Reply:
That was the reason for having "Benefit" for "None" and "Benefit / {type}" for the others

Reply:
Hm what about "Basic" if it's none.. 🤔 perhaps we could then just use "Basic" as the value as well.

Reply:
That works :+1: can always adjust the wording later